types/Cargo.toml

```
@@ -4,7 +4,7 @@ version = "0.3.0"
 edition = "2021"
 
 [dependencies]
-eth-types = { git = "https://github.com/kroma-network/zkevm-circuits.git", rev = "4f431bf", features = [
+eth-types = { git = "https://github.com/kroma-network/zkevm-circuits.git", rev = "f534f0e", features = [
```

Please apply this to else where.

```suggestion
eth-types = { git = "https://github.com/kroma-network/zkevm-circuits.git", tag = "v0.2.0", features = [
```